skills/health-check/SKILL.md

```
@@ -0,0 +1,255 @@
+  - jens.reynders@novemberfive.co
+tags:
+  - engineering
+---
+
+# Health Check
+
+Perform a structured project health assessment following the company's standardised process. Produces a comprehensive report with RAG status and prioritised action items.
+
+## Workflow Overview
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)